data/saas/config/openweb_config.yml

```
@@ -0,0 +1,64 @@
+# Notes about the test request below  
+  # Normally we would not hardcode a value here, however, we won't know the  value ahead of time, and we only need to verify the SSO token is valid. This call will return a 404 if the primary_key is not found. A 401 will be returned if the SSO token is invalid.
+
+saas_config:
+  fides_key: <instance_fides_key>
+  name: OpenWeb
+  type: openweb
+  description: A sample schema representing the OpenWeb integration for Fides
+  user_guide: https://docs.ethyca.com/user-guides/integrations/saas-integrations/openweb
+  version: 0.1.0
+
+  connector_params:
+    - name: domain
+      label: Domain (default www.spot.im)
+      default_value: www.spot.im
+    - name: api_key
+      label: SSO Authentication Token
+      description: Value of the OpenWeb SSO Authentication token. This can be found in your OpenWeb admin GUI by going to 'Settings' then navigate to the Authentication Tokens.
+      sensitive: True
+    - name: x_spot_id
+      label: Your SPOT_ID
```

```suggestion
      label: Spot ID
```
Make sure to use the naming convention that OpenWeb uses in their docs.